getting-started.md

```diff
@@ -0,0 +1,125 @@
+
+### Resources
+* Rule examples: 
+  * [github.com/testthedocs](https://github.com/testthedocs/vale-styles/tree/master/ttd-light)
+  * [github.com/errata-ai/vale](https://github.com/errata-ai/vale/tree/v3/testdata/styles)
+
+* Documentation:
+  * In general, the two most useful pages from the official Vale documentation for rule development are the [Styles](https://vale.sh/docs/topics/scoping/) and [Scoping](https://vale.sh/docs/topics/scoping/) references.
+
+## Regex
+Sometimes you'll need to search for specific patterns, expressions, or symbols instead of single words. This is where regular expressions come in.
+
+For example, to search for the expression "Not only ...but also", you can define a token that will capture this regardless of what is in between. 
+```yaml
+tokens:
+  - 'not only.*?but also'
+```
+The single quotes indicate the beginning and end of the regex, and the `.*?` quantifier indicates that any characters are allowed in between when looking for matches.
+
+### Resources
+- [Interactive regex tutorial](https://regexone.com/): Great for  beginners
+- [Regex editor](https://regex101.com/): Online tool that helps you compose and test your expressions. I like how it highlights capture groups in different colors.
+- Before composing an insane regex, remember to check for [existing rules](#more-rule-examples) that migh've done this already. 
+
+
+## Test rules
+Create a file in the root directory of the repository and fill it with text that you expect your rule to catch. It can be in any text format (`.txt`, `.md`, `.rst`...)
+
+Besides the text file, the `vale` command needs a `vale.ini` config file. This file already exists in the root folder `praecepta/`. Vale will automatically find the `vale.ini` if you run the command in the same directory.
+
+### Test all rules
+To apply all the rules inside `styles/Canonical` at the same time, run `vale <file>` in the root directory `praecepta/`.
+
+Example:
+```shell
+vale test.md
+```
+
+### Test a specific rule
+To limit the test to a specific rule, you have to modify the `vale.ini` config file's [style specification](https://vale.sh/docs/topics/config/#basedonstyles).
+
+The default config specifies `BasedOnStyles = Canonical`, which will test all rules in that folder.
+
+Comment that line out and add a line enabling your rule using the syntax `<style>.<rule> = <YES | NO>`
+
+
+This edit would look similar to the output below:
+```ini
+; BasedOnStyles = Canonical
+Canonical.000-My-Rule = YES
+```
+
+Once it is saved, you can run the command `vale <file>` in the root directory `praecepta/`.
+
+Example:
+```shell
+vale test.md
+```
+in the root directory `praecepta/`.
+
+### Resources
+* [Configuration documentation](https://vale.sh/docs/topics/config/#basedonstyles) -  make your own `vale.ini` file(s) with specific configs for your tests
+* Run `vale -h` to see optional parameters like output style and config path.
+
+## Troubleshooting
+### My rule isn't getting applied to the text
```

Comment:
Since you've only got one point under the troubleshooting, a level 3 subheading feels a bit redundant (it would be fine just as text)

Reply:
I originally added it so that it would appear in the ToC, but I think you're right - just one "Troubleshooting" heading is enough for you to see where to go if your rule ain't working.